File: Service/Implement/CauHoiATTPQuestionsService.cs
namespace Service.Implement
{
    public class CauHoiATTPQuestionsService : BaseService<CauHoiATTPQuestions, ICauHoiATTPQuestionsRepository>
    , ICauHoiATTPQuestionsService
    {
    private readonly ICauHoiATTPQuestionsRepository _CauHoiATTPQuestionsRepository;
    public CauHoiATTPQuestionsService(ICauHoiATTPQuestionsRepository CauHoiATTPQuestionsRepository) : base(CauHoiATTPQuestionsRepository)
    {
    _CauHoiATTPQuestionsRepository = CauHoiATTPQuestionsRepository;
    }
    }
    }

File: CMS/src/app/shared/CauHoiATTPQuestions.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { CauHoiATTPQuestions } from 'src/app/shared/CauHoiATTPQuestions.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class CauHoiATTPQuestionsService extends BaseService{

    List: CauHoiATTPQuestions[] | undefined;
    ListFilter: CauHoiATTPQuestions[] | undefined;
    FormData!: CauHoiATTPQuestions;    

    DisplayColumns001: string[] = ['Name', 'SortOrder', 'is_ans', 'Save'];

    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "CauHoiATTPQuestions";
    }
}

